src/main/java/com/hyunec/cosmicbaseball/controller/BaseballController.java

```diff
@@ -2,31 +2,42 @@
 
 import com.hyunec.cosmicbaseball.entity.BaseballGame;
 import com.hyunec.cosmicbaseball.handler.ResultHandler;
+import com.hyunec.cosmicbaseball.util.BattingResult;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import javax.xml.transform.Result;
-
 @RestController
 public class BaseballController {
 
-    private final BaseballGame baseballGame;
-    private final ResultHandler resultHandler;
+  private final Logger logger = LoggerFactory.getLogger(BaseballController.class);
```

Comment:
```suggestion
  private final Logger logger = LoggerFactory.getLogger(this.getClass());
```

`nit` 이렇게 하면 복붙해서 사용할 수 있습니다.

Reply:
이렇게하면 페이지마다 로거 생성 안해도 그냥 logger.debug 해서 쓸 수 있다는 거죠 ??
진짜 처음 알았어요짱..